### PR TITLE
luggage rack

### PR DESCRIPTION
**Pull request by AlisGraveNil**
## Description
This is a 13-key macropad. It's the 3-wide nav cluster between the alphanumerics and the numpad.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [X] Keyboard (addition or update)
- [X] Keymap/layout/userspace (addition or update)
- [X] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [X] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [X] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [X] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [X] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [X] I have added tests to cover my changes.
- [X] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — CHANGES_REQUESTED** [review]
You may need to rebase to fix the git submodules.

**Comment by AlisGraveNil** [comment]
@drashna - I think the rebase worked

**Review by drashna — APPROVED** [review]
Yup! 

LGTM.

File: keyboards/clawsome/luggage_rack/info.txt [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
```
Comment (fauxpark) [inline keyboards/clawsome/luggage_rack/info.txt:1]: Wrong filetype, should be `info.json`.

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Comment by fauxpark** [comment]
Also, missing a readme.

**Comment by AlisGraveNil** [comment]
@fauxpark - I've uploaded the readme

File: keyboards/clawsome/luggage_rack/info.txt (hunk 1 above)
Comment (fauxpark) [reply keyboards/clawsome/luggage_rack/info.txt:1]: Still outstanding.

Comment (AlisGraveNil) [reply keyboards/clawsome/luggage_rack/info.txt:1]: tried to adjust from .txt to .json, ended up pulling a bunch of random files during a rebase.

what should I do?

**Comment by AlisGraveNil** [comment]
@drashna How do I make this PR un-pull those thousands of files?

**Comment by AlisGraveNil** [comment]
@fauxpark okay - excess files removed, readme added

File: keyboards/clawsome/luggage_rack/info.txt (hunk 1 above)
Comment (fauxpark) [reply keyboards/clawsome/luggage_rack/info.txt:1]: Just rename the file? You don't need to do any rebasing.

Comment (AlisGraveNil) [reply keyboards/clawsome/luggage_rack/info.txt]: It did that thing where my local GitHub Desktop decided to "adjust" some 1900 files and count that towards my file count. I made it undo that today.

Apparently I also had an 'info.txt' and an 'info.json' in there, so I just deleted the former.

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**